flocker/node/_change.py

```diff
@@ -77,7 +84,10 @@ def run_state_change(change, deployer):
             )
         return d
 
-    return change.run(deployer)
+    with change.eliot_action.context():
```

Comment:
Yea, seems reasonable.  It's a lot of boilerplate for "run this async function in the context of this action".

Filed https://github.com/ClusterHQ/eliot/issues/158